nucleus/parent/pom.xml

```diff
@@ -94,6 +94,8 @@
         <jakarta.inject-api.version>2.0.1.MR</jakarta.inject-api.version>
         <hk2.version>3.1.0</hk2.version>
         <hk2.plugin.version>3.1.0</hk2.plugin.version>
+<!--        <hk2.version>3.1.1-SNAPSHOT</hk2.version>
+        <hk2.plugin.version>3.1.1-SNAPSHOT</hk2.plugin.version>-->
```

Comment:
Now master depends on HK2 3.1.1 SNAPSHOT. We have to release it asap.

Reply:
The snapshot version is commented out, so no harm. I forgot to remove this. I think it's OK to remove this comment when we upgrade hk2 in the future.

Reply:
Aha, I did not notice the comment goes through two lines :-)